### DISCUSSION TITLE
How to use the "pagination" feature?

### DISCUSSION THREAD

**Discussion by pglizniewicz** [WerWolv/ImHex#1080] [Get Help] [answered]
How can I set the "page" size? When I open a file, I always see "Page 1/1". I would like to see my file (a RAW CD dump) split into blocks so I can view the data on the "sector level".

**Comment by WerWolv** [comment, marked as answer]
Hey! Pages are currently hardcoded to 0x1000'0000 bytes / 256MiB. They were introduced mainly as a workaround for an ImGui issue (the UI library ImHex uses). Making them user definable in the future (as long as they are <= 0x1000'0000 bytes in length) should be possible though.
